addon/components/auto-complete.js

```diff
@@ -5,9 +5,10 @@ export default Ember.Component.extend({
   hightlightIndex: -1,
   escapedChars: [40, 38, 13],
   visibility: "display:none;",
-  inputClazz: function() {
+  inputClass: '',
```

Comment:
I saw an undefined in the class all the time, I suppose the reason is that the property wasn't defined.